sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -304,8 +304,9 @@ object ScalaJSPluginInternal {
         val output = (artifactPath in key).value
         val linker = (scalaJSLinker in key).value
         val usesLinkerTag = (usesScalaJSLinkerTag in key).value
+        val scalaJSLinkTag = ScalaJSTags.Link
```

Comment:
I'm not sure this `val` is necessary. I suggest directly using `ScalaJSTags.Link` below. It also helps keeping that series of `val`s dedicated to all the `.value`s.